base/uk.ac.stfc.isis.ibex.ui.devicescreens/src/uk/ac/stfc/isis/ibex/ui/devicescreens/models/EditDeviceScreensDescriptionViewModel.java

```diff
@@ -383,16 +373,9 @@ public String getCurrentKey() {
      *            the new value to set
      */
     public void setCurrentKey(String key) {
```

Comment:
Other methods have lost their "Current". This probably should too unless there's a good reason not to